### DISCUSSION TITLE
SAE eingerichtet, doch leider keine Leistungswerte im Sunny Portal

### DISCUSSION THREAD

**Discussion by NHJohnD** [camueller/SmartApplianceEnabler#496] [Q&A]
Hallo,

nach langen hin und her habe ich es endlich geschafft, das der SAE und das Gerät im Sunny Portal angezeigt werden. Doch leider zeigt er mir nicht die Leistungsdaten an, es steht also immer 0 Watt da. 

Die Einstellungen habe ich als Bild mit eingefügt. Was mache ich jetzt noch falsch?

Wäre sehr über eure Hilfe dankbar. 

<img width="1316" alt="Bildschirm­foto 2023-03-09 um 11 09 28" src="https://user-images.githubusercontent.com/121162089/223990324-58d87218-e798-4f40-ae4b-1d1400fc2acf.png">

**Comment by camueller** [comment]
Die Frage kam schon oft: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/QA_DE.md

**Reply by NHJohnD** [reply]
Danke für die schnelle Antwort...

Leider muss ich diese Frage stellen, wo finde ich den DeviceStatus?

**Reply by camueller** [reply]
Das steht eine Zeile darüber, bitte richtig lesen.
